schema/Query.graphql

```diff
@@ -28,4 +28,14 @@ type Query {
     Generate challenge to authenticate with WebAuthn
     """
     generateAuthenticatorChallenge(username: String!): AuthenticationWithWebPublicKeyCredential
+
+    """
+    Get a life by its ID
+    """
+    getLife(id: ObjectID!): Life!
```

Comment:
Think `Life` is not mandatory as it will be possible that a Life with this `id` does not exist